### ISSUE TITLE
Unable to open files via command line

### ISSUE THREAD

**Issue by Raph-2018** [OpenShot/openshot-qt#2162]
Hello,
I installed the latest version (2.4.3), and I noticed the following bug: files, either .osp or any media files, can not be opened in Openshot from any file manager nor from the terminal. It seems to be a regression, as it was working perfectly with the previous version (2.4.2).

System details: Ubuntu 14.04, Openshot 2.4.3.

To Reproduce:
1) Create a project file 'project.osp' with Openshot
2) In a terminal, run: openshot-qt /path/to/project.osp
→ expected result: openshot should open the file
→ what I get: Openshot does not open, with the message: "openshot-qt: error: unrecognized arguments"

It does not work even with a path without spaces or special characters. 
Furthermore, files cannot be opened from the file manager. But I can open the files successfully from within Openshot, using the menu. So that it seems that the problem comes from the command line argument for opening files!

**Comment by Raph-2018** [comment]
@DylanC - I updated from Openshot 2.4.2 with Ubuntu's built-in update software.
I added (a long time ago) the ppa: http://ppa.launchpad.net/openshot.developers/ppa/ubuntu trusty main

I notice that I have been a bit vague in my first post: my Ubuntu flavor is actually xubuntu 14.04 LTS.

**Comment by DylanC** [comment]
@Raph-2018 - I can confirm this even on Ubuntu 18.04 too. Thanks.

**Comment by DylanC** [comment]
Seems to be a dup of #1872.

**Comment by Raph-2018** [comment]
I just installed the new release 2.4.4, and this issue seems to have been improperly fixed. Here are the problems I noticed:
1) If I open a .osp file from the file manager, thumbnails of videos are not displayed on the timeline.
2) If I open a media file with Openshot from the file manager, and then I try to cut it on the timeline, a gap appears between the edges of the cutted video and what is displayed on the video preview window. Strangely enough, there is no such behaviour when the media file is opened from inside Openshot. I have tried with different kinds of videos and that always happened, but the gap seems to be more or less important depending of the video length.
